localization/i18n/fr/OrcaSlicer_fr.po

```diff
@@ -7212,34 +7186,34 @@ msgid ""
 "Are you sure you want to update? This will take about 10 minutes. Do not "
 "turn off the power while the printer is updating."
 msgstr ""
-"Êtes-vous sûr de vouloir mettre à jour ? Cela prendra environ 10 minutes. Ne "
-"coupez pas l'alimentation pendant la mise à jour de l'imprimante."
+"Êtes-vous sûr de vouloir effectuer la mise à jour ? Cela prendra environ 10 "
+"minutes. Ne mettez pas l'imprimante hors tension durant la mise à jour."
 
 msgid ""
 "An important update was detected and needs to be run before printing can "
 "continue. Do you want to update now? You can also update later from 'Upgrade "
 "firmware'."
 msgstr ""
-"Une mise à jour importante a été détectée et doit être exécutée avant que "
-"l'impression puisse continuer. Voulez-vous mettre à jour maintenant ? Vous "
-"pouvez également mettre à jour plus tard via le bouton ‘Mettre à jour le "
-"firmware'."
+"Une mise à jour importante a été détectée et doit être exécutée avant de "
+"pouvoir poursuivre l'impression. Voulez-vous effectuer la mise à jour "
+"maintenant ? Vous pouvez également effectuer une mise à jour ultérieurement "
+"à partir de \"Mettre à jour le firmware\"."
 
 msgid ""
 "The firmware version is abnormal. Repairing and updating are required before "
 "printing. Do you want to update now? You can also update later on printer or "
 "update next time starting the studio."
 msgstr ""
-"La version du firmware est anormale. Une réparation et une mise à jour sont "
-"nécessaires avant l'impression. Voulez-vous mettre à jour maintenant ? Vous "
-"pouvez également mettre à jour plus tard via l'imprimante ou au prochain "
-"démarrage de Orca Slicer."
+"La version du firmware est erronée. La réparation et la mise à jour sont "
+"nécessaires avant l'impression. Voulez-vous effectuer la mise à jour "
+"maintenant ? Vous pouvez également effectuer une mise à jour ultérieurement "
+"depuis l'imprimante ou lors du prochain démarrage de Bambu Studio."
```

Comment:
I think we shouldn't copy BS's translation.

Reply:
Good catch ! Fixed !
However, line 6767, there is a mention to "Bambu Studio" in the original text.
I only merged strings that were in common, ignored string that are not in Orca, so it shouldn't be problematic for the file itself